docker/dockerfile-build-fedora

```diff
@@ -11,40 +9,39 @@ ARG user_uid
 # Install dependent packages
 # Dependencies:
 # * hcc-config.cmake: pkg-config
-# * tensile: python2.7, python-yaml
-# * rocblas-test: gfortran, googletest
-# * rocblas-bench: libboost-program-options-dev
-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+# * tensile: python34, PyYAML
+# * rocblas-test: gcc-gfortran, googletest
+# * rocblas-bench: boost-devel
+RUN dnf -y update && dnf install -y \
     sudo \
-    build-essential \
-    cmake \
-    clang-format-3.8 \
     ca-certificates \
     git \
-    pkg-config \
-    python2.7 \
-    python-yaml \
-    gfortran \
-    libboost-program-options-dev \
-    rpm \
+    make \
+    cmake \
+    clang \
+    gcc-c++ \
+    gcc-gfortran \
+    python34 \
+    PyYAML \
```

Comment:
A question: is the change from python2.7 to python34 for "non Tensile" parts of the code? Does Tensile still use python2.7?

Reply:
Yes, I needed it for some other dependency, but unfortunately I can't remember which one anymore.  I believe that Fedora already has python2 installed.